data/disney

```diff
@@ -47,14 +47,10 @@ disney.sg
 20thcenturystudios.com.au
 20thcenturystudios.com.br
 20thcenturystudios.jp
-adobedtm.com
 adventuresbydisney.com
 babble.com
 babyzone.com
-bamgrid.com
```

Comment:
Sorry, I means `bam.nr-data.net` is not owned by Disney, but `bamgrid.com` is. It should be readded.